src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -29,33 +40,42 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
   def withActiveUser(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
       getActiveSamUser(oidcHeaders, userService, termsOfServiceConfig, samRequestContext).unsafeToFuture()
-    }.tmap { samUser =>
+    }.flatMap { samUser =>
       logger.debug(s"Handling request for active Sam User: $samUser")
-      samUser
+      logRequestResultWithSamUser(samUser, oidcHeaders)
     }
   }
 
   def asAdminServiceUser: Directive0 = requireOidcHeaders.flatMap { oidcHeaders =>
-    Directives.mapInnerRoute { r =>
-      if (!adminConfig.serviceAccountAdmins.contains(oidcHeaders.email)) {
-        reject(AuthorizationFailedRejection)
-      } else {
-        logger.info(s"Handling request for service admin account: ${oidcHeaders.email}")
-        r
+    Directives
+      .mapInnerRoute { r =>
+        if (!adminConfig.serviceAccountAdmins.contains(oidcHeaders.email)) {
+          reject(AuthorizationFailedRejection)
+        } else {
+          logger.info(s"Handling request for service admin account: ${oidcHeaders.email}")
+          r
+        }
       }
-    }
+      .tflatMap(_ => logAdminServiceAdminUserRequestResult(oidcHeaders))
   }
 
   def withUserAllowInactive(samRequestContext: SamRequestContext): Directive1[SamUser] = requireOidcHeaders.flatMap { oidcHeaders =>
     onSuccess {
       getSamUser(oidcHeaders, userService, samRequestContext).unsafeToFuture()
-    }.tmap { samUser =>
+    }.flatMap { samUser =>
       logger.debug(s"Handling request for (in)active Sam User: $samUser")
-      samUser
+      logRequestResultWithSamUser(samUser, oidcHeaders)
```

Comment:
This one catches requests by inactive users.

Reply:
Is there a way to differentiate between active/inactive in the logs? Maybe we just want to leave this one out for now if they wont count for MAU/MEU?